include/ideep/abstract_types.hpp

```diff
@@ -70,13 +83,20 @@ static bool has_bf16_type_support() {
   // static bool support_bf16 = isa >= dnnl::cpu_isa::avx512_core
   //                           && isa != dnnl::cpu_isa::avx2_vnni;
   static bool support_bf16 =
-      dnnl::get_effective_cpu_isa() >= dnnl::cpu_isa::avx512_core;
+      dnnl::get_effective_cpu_isa() >= dnnl::cpu_isa::avx2_vnni_2;
   return support_bf16;
 }
 
+static bool check_isa_avx2_vnni_2() {
+  static bool is_avx2_vnni_2 =
+      dnnl::get_effective_cpu_isa() == dnnl::cpu_isa::avx2_vnni_2;
+  return is_avx2_vnni_2;
+}
```

Comment:
What's the intention of this API? Checking if the CPU supports avx2 vnni 2 and not others? If so, should we rename the function as `check_isa_is_avx2_vnni_2`?

Reply:
Fixed